staging/src/k8s.io/apiserver/pkg/endpoints/filters/limit_request.go

```diff
@@ -57,7 +58,7 @@ func WithLimitUntilHealthy(handler http.Handler, c *rest.Config) http.Handler {
 		if !healthyFirstTime {
 			var code int
 			coreClient.RESTClient().Get().AbsPath("/healthz").Do(context.TODO()).StatusCode(&code)
-			if code != http.StatusInternalServerError {
+			if code != http.StatusInternalServerError || code != http.StatusTooManyRequests {
```

Comment:
`code != http.StatusInternalServerError || code != http.StatusTooManyRequests`  is always true ?  should use  ` && `

Reply:
Shouldn’t code always be 200?

Reply:
Alright. I will check for 200.